doc/dev/documenting.rst

```diff
@@ -2,6 +2,20 @@
  Documenting Ceph
 ==================
 
+User documentation
+==================
+
+The documentation on docs.ceph.com is generated from the restructuredText
+sources in ``/doc/`` in the Ceph git repository.
+
+Please make sure that your changes are written in a way that is intended
+for end users of the software, unless you are making additions in
+``/doc/dev/``, which is the section for developers.
+
+All pull requests that modify user-facing functionality must
+include corresponding updates to documentation: see 
+`Submitting Patches`_ for more detail.
+
```

Comment:
I think we want this here in the procedures for developers so that code contributors see it, not just people seeking out info about documentation.  However, I'll add a hyperlink here to the start/documenting-ceph page, so that we're not tempted to duplicate too much stuff.